pkgs/applications/window-managers/i3/altlayout.nix

```
@@ -0,0 +1,22 @@
+{ lib, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "i3altlayout";
+  version = "0.3";
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "1h0phf3s6ljffxw0bs73k041wildaz01h37iv5mxhami41wrh4qf";
+  };
+
+  nativeBuildInputs =  with python3Packages; [ enum-compat i3ipc docopt ];
+  pythonPath = with python3Packages; [ enum-compat i3ipc docopt ];
+
+  meta = with lib; {
+    maintainers = with maintainers; [ magnetophon ];
+    description = "helps you handle more efficiently your screen real estate in i3wm by auto-splitting windows on their longest side.";
```

```suggestion
    description = "Helps you handle more efficiently your screen real estate in i3wm by auto-splitting windows on their longest side";
```

See https://nixos.org/manual/nixpkgs/unstable/#sec-standard-meta-attributes